src/tests/class_tests/openms/source/TMTThirtyTwoPlexQuantitationMethod_test.cpp

```diff
@@ -0,0 +1,308 @@
+// Copyright (c) 2002-present, OpenMS Inc. -- EKU Tuebingen, ETH Zurich, and FU Berlin
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// --------------------------------------------------------------------------
+// $Maintainer: Chris Bielow $, Julianus Pfeuffer $
+// $Authors: Chris Bielow $
+// --------------------------------------------------------------------------
+
+#include <OpenMS/CONCEPT/ClassTest.h>
+#include <OpenMS/test_config.h>
+
+///////////////////////////
+#include <OpenMS/ANALYSIS/QUANTITATION/TMTThirtyTwoPlexQuantitationMethod.h>
+///////////////////////////
+
+#include <OpenMS/DATASTRUCTURES/Matrix.h>
+
+using namespace OpenMS;
+using namespace std;
+
+START_TEST(TMTThirtyTwoPlexQuantitationMethod, "$Id$")
+
+/////////////////////////////////////////////////////////////
+/////////////////////////////////////////////////////////////
+TMTThirtyTwoPlexQuantitationMethod* ptr = nullptr;
+TMTThirtyTwoPlexQuantitationMethod* null_ptr = nullptr;
+
+START_SECTION(TMTThirtyTwoPlexQuantitationMethod())
+{
+  ptr = new TMTThirtyTwoPlexQuantitationMethod();
+    TEST_NOT_EQUAL(ptr, null_ptr)
+}
+END_SECTION
+
+START_SECTION(~TMTThirtyTwoPlexQuantitationMethod())
+{
+  delete ptr;
+}
+END_SECTION
+
+START_SECTION((const String& getMethodName() const ))
+{
+  TMTThirtyTwoPlexQuantitationMethod quant_meth;
+  TEST_EQUAL(quant_meth.getMethodName(), "tmt32plex")
+}
+END_SECTION
+
+START_SECTION((const IsobaricChannelList& getChannelInformation() const ))
```

Comment:
please add a test to IsobaricChannelExtractor_test for 32 / or 35 plex using real data, to make sure that the values are extracted as planned.
Either use artificial data or the mzML I provided - for the latter, manually check the expected intensities.